addons/mail/static/src/core/web/discuss_patch.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
     <t t-inherit="mail.Discuss" t-inherit-mode="extension">
         <xpath expr="//*[@t-ref='mobileTopbar']" position="before">
-            <DiscussSidebar t-if="!ui.isSmall"/>
+            <DiscussSidebar t-if="!ui.isSmall and !thread?.readonly"/>
```

Comment:
I think it would be nice to always keep it, also if we need to hide it, it's really strange to condition the sidebar to the currently displayed thread.